fe1-web/package.json

```diff
@@ -5,6 +5,8 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
+    "prebuild-web": "node_modules/expo-optimize/build/index.js",
+    "build-web": "expo build:web",
```

Comment:
@Xelowak @jiabaow 
FYI, "pre" and "post" are prefixes that can be used to execute some operation before or after a given command.
So `npm run build-web` will actually execute:
- prebuild-web
- build-web
- postbuild-web ( which doesn't exist, so it won't be run )

Reply:
Okay, I didn't know that